scenes/game/game.gd

```diff
@@ -2891,6 +2904,8 @@ func _handle_events(events):
 				delay = _on_damage(event)
 			Enums.EventType.EventType_Strike_WildStrike:
 				_on_strike_started(event, false)
```

Comment:
what if you did an ex strike that you can't pay then becomes wild? You would need to get rid of your current EX Strike thing.
Another reason to maybe have it displayed on the Card object somehow.